src/jsutils.h

```diff
@@ -57,21 +60,30 @@ size_t flash_strlen(const char *str);
 /// Copy a string from flash to RAM
 char *flash_strncpy(char *dest, const char *source, size_t cap);
 
+/// Compare a string in memory with a string in flash
+int flash_strcmp(const char *mem, const char *flash);
```

Comment:
The only place flash_strcmp is called is in jswrapper.c. In the current way that's written there is no extra ifdef because of it. But if I reorg the #define above may be useful.

What do you think of when you read "safe_strcmp"? Safe from what? Interrupts? Buffer overruns?